.github/workflows/linkcheck.yml

```diff
@@ -0,0 +1,61 @@
+name: Check URL Status
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: 0 0 * * 0 # run at midnight every Sunday
+
+env:
+  LINK_TRACKING_ISSUE_NUMBER: 85
```

Comment:
So this Issue should then never be closed? Or will it automatically reopen?

Reply:
It will automatically reopen when there are broken URLs, and automatically close when there are no broken URLs*. Feel free to tweak the workflow if you want it to remain closed.

*that was the intention at least. I don't know if that functionality works 100% (mainly because I don't know how Lychee reports "unknown" or links of other classes). Its a small point though.